.vscode/launch.json

```diff
@@ -30,5 +30,12 @@
       "program": "${workspaceFolder}/cmd/spice/main.go",
       "args": ["version"]
     },
+    {
+      "type": "lldb",
+      "request": "attach",
+      "name": "[Rust] Attack to spiced",
+      "program": "~/.spice/bin/spiced",
+      "pid": "" // pid of the process use `pgrep spiced`
```

Comment:
You can use input variables to prompt user for pid
https://code.visualstudio.com/docs/editor/variables-reference#_input-variables